tasks/plugins/aiida-quantumespresso.yml

```diff
@@ -4,6 +4,7 @@
     name: aiida-quantumespresso
     version: "{{ aiida_quantumespresso_version }}"
     virtualenv: "{{ aiida_venv }}"
+    extra_args: "--pre"
```

Comment:
very strange, so without `--pre` it was installing a *different* version from the one we specified?

Reply:
seems to be the case...indeed...